### PR TITLE
Integrate-upstream-modal-updates

### PR DESCRIPTION
**Pull request by PavelLaptev**
### Pull Request Description

**Title:** Integrate upstream modal updates

**Description:**

This pull request includes several updates and enhancements to the integration modal functionality:

1. **Fixes:**
   - Bolded the target branch name in the divergence message to improve visibility and clarity for users.

2. **Features:**
   - Updated the `IntegrationSeriesRow` component to utilize series data instead of just the title, allowing for a more comprehensive display of information.
   - Added a tooltip for additional series in the labels row, enhancing the user interface by providing more context and information at a glance.

These changes aim to improve the user experience and provide clearer information regarding integration series and their statuses. Please review the modifications and provide any feedback or suggestions.

### CODE REVIEW

**Review by krlvi — APPROVED** [review]
lgtm

File: apps/desktop/src/lib/components/IntegrateUpstreamModal.svelte [hunk 1]
```diff
@@ -167,7 +167,7 @@
 				<div class="target-divergence-about">
 					<h3 class="text-14 text-semibold">Target branch divergence</h3>
 					<p class="text-12 text-body target-divergence-description">
-						Branch target/main has diverged from the workspace.
+						<span class="text-bold">target/main</span> has diverged from the workspace.
```
Comment (mtsgrd) [inline apps/desktop/src/lib/components/IntegrateUpstreamModal.svelte:170]: Should we be getting this "target/main" from the baseBranchService perhaps?

**Review by mtsgrd — APPROVED** [review]

**Comment by krlvi** [comment]
@PavelLaptev what do you think about moving the tag so that things are aligned? 
image

**Comment by PavelLaptev** [comment]
> @PavelLaptev what do you think about moving the tag so that things are aligned?
Sure, just pushed the commit 
image

---

I also updated it for V3. On V3 it should look like this 
image

**Pull request merged**